tests/phpunit/tests/Discovery.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+namespace Google\Web_Stories\Tests;
+
+class Discovery extends \WP_UnitTestCase {
```

Comment:
```suggestion
/**
 * @coversDefaultClass \Google\Web_Stories\Discovery
 */
class Discovery extends \WP_UnitTestCase {
```

Then you don't have to repeat it for all the `@covers` annotations below

Reply:
(As per https://phpunit.readthedocs.io/en/7.3/annotations.html#coversdefaultclass)

Reply:
We should add these everywhere.